data/members/judy.json

```
@@ -0,0 +1,5 @@
+{
+  "name":"judy"
+  "github":"judysohn4"
```

```suggestion
  "name":"judy",
  "github":"judysohn4",
```
make sure lines are separated with commas